manifests/init.pp

```diff
@@ -19,6 +19,9 @@
 # $proxy_password::     Proxy password for authentication
 #                       type:String
 #
+# $pulp_max_speed::     Maximum bandwidth used per download in kb/sec per worker
```

Comment:
Actually this isn't quite correct. ```to_bytes``` without a suffix is still in bytes, but it also converts kB, MB, GB. It looks case sensitive. It's also the SI KB so 1000 bytes rather than the KiB which is 1024 bytes. Not sure how much detail you need to go into, but I can see users being surprised by this.